Sources/Testing/ABI/v0/Encoded/ABIv0.EncodedBacktrace.swift

```diff
@@ -0,0 +1,42 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+extension ABIv0 {
+  /// A type implementing the JSON encoding of ``Backtrace`` for the ABI entry
+  /// point and event stream output.
+  ///
+  /// This type is not part of the public interface of the testing library. It
+  /// assists in converting values to JSON; clients that consume this JSON are
+  /// expected to write their own decoders.
+  struct EncodedBacktrace: Sendable {
```

Comment:
Can/should we document this new encoding type in the ABI documentation schema?

Reply:
This remains experimental and underscored. We aren't ready to promote it to API.